catalog/aws.go

```diff
@@ -95,23 +112,47 @@ func (a *aws) transformServices(aservices []sd.ServiceSummary) map[string]servic
 	return services
 }
 
+func (a *aws) transformNamespace(anamespace *sd.Namespace) namespace {
+	namespace := namespace{id: *anamespace.Id, name: *anamespace.Name}
```

Comment:
nit: I find the variable names here kinda hard to follow on first reading! `anamespace` I thought was a typo at first until I realised it's trying to avoid colliding with the internal `namespace`. Although it's technically not a problem I also found it hard to parse at first that `namespace` is both a variable and a type name in this line.

I'd be tempted to call the argument `ns` and the internal one `n` or something. Kinda arbitrary but easier to distinguish within this short method IMO.

Reply:
`anamespace` is also a shorthand for `awsnamespace` because it is the one coming from the sdk in contrast to the other one.

Reply:
I changed it to `awsNamespace` here and in other places too.